scanf.c

```diff
@@ -146,12 +164,16 @@ RETRACE_REPLACE(vsscanf)
 int
 RETRACE_IMPLEMENTATION(vfscanf)(FILE *stream, const char *format, va_list ap)
 {
-	rtr_vfscanf_t real_vfscanf = RETRACE_GET_REAL(vfscanf);
-	rtr_fileno_t fileno_ = RETRACE_GET_REAL(fileno);
+	rtr_vfscanf_t real_vfscanf;
+	rtr_fileno_t fileno_;
 	va_list ap1;
+	int result;
+
+	real_vfscanf = RETRACE_GET_REAL(vfscanf);
+	fileno_ = RETRACE_GET_REAL(fileno);
 
-	va_copy(ap1, ap);
-	int result = real_vfscanf(stream, format, ap);
+	__va_copy(ap1, ap);
```

Comment:
va_copy() occurs compile error in current Makefile (I think C90), so I replaced it to \_\_va_copy().